auxx/src/Mode.hs

```diff
@@ -79,6 +81,43 @@ data CmdCtx = CmdCtx
     }
 
 type AuxxMode = ReaderT AuxxContext Production
+type MonadAuxxMode m =
```

Comment:
Nice constraint list. Initially I thought to suggest replacing it with `WorkMode`, but it looks like it's actually a bit smaller (doesn't include `Slog`, for instance), so let's leave it as is.

Reply:
For `Slog` there are only `Has` classes, so it's not required because this list contains `MonadReader AuxxContext m` :)
Also it looks a bit weird, because if you have `MonadReader AuxxContext` and some basic stuff which `Production` has, you have all other classes from this list.